packages/utils/styling/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@fluentui-react-native/styling-utils",
+  "version": "0.1.0",
```

Comment:
Everytime we add a new package, the matrix of dependencies gets slightly more complex, which also means more change files and more things for clients to update. I thought this mores when I was thinking about "what is the minimum set of things we could put into the platform bundle?". While I wouldn't block this review on it, it might be worth seeing if we could reconcile a bunch of these packages we have together in the future.

Reply:
Would you mind opening an issue with this thought? Tagging pat/sammy/krystal in it would be good, too, since it has to do with platform bundle complexity

Reply:
That sounds good! I shall do that later today =)